test/tests_common.py

```diff
@@ -219,6 +219,36 @@ def test_common_tensorset_multiproc(env):
     env.assertEqual(values, [b'2', b'3'])
 
 
+def test_common_tensorset_multiproc_blob(env):
```

Comment:
@lantiga you can quickly check the improvements running this test against master. If we use a larger tensor the improvements should be even larger.